apps/gp2-server/src/data-providers/project.data-provider.ts

```diff
@@ -107,6 +111,29 @@ export function parseProjectToDataObject({
   if (project.keywords && !project.keywords.every(gp2.isProjectKeyword)) {
     throw new TypeError('Invalid keyword received from Squidex');
   }
+  const milestones =
+    project.milestones?.reduce(
```

Comment:
why exactly is this a reduce, not a map?

Reply:
done.